Library/Formula/iojs.rb

```diff
@@ -0,0 +1,38 @@
+class Iojs < Formula
+  homepage "https://iojs.org/"
+  url "https://iojs.org/dist/v1.0.4/iojs-v1.0.4.tar.xz"
+  sha256 "c902f5abbd59c56346680f0b4a71056c51610847b9576acf83a9c210bf664e98"
+
+  keg_only "`iojs` symlinks conflict with `node` but can be used by prepending your PATH"
+
+  option "with-debug", "Build with debugger hooks"
+
+  depends_on :python => :build
+
+
```

Comment:
Blank line here needs to be removed. Otherwise, Bot reports all is good.

```
==> brew audit iojs --strict						 FAILED
==> brew style iojs
== /usr/local/Library/Formula/iojs.rb ==
C: 12:  1: Extra blank line detected.

1 file inspected, 1 offense detected
```